src/js/facility-locator/components/MapboxClient.js

```diff
@@ -0,0 +1,6 @@
+import MapboxClient from 'mapbox';
+
+export const mapboxToken = 'pk.eyJ1IjoiYXlhbGVsb2VociIsImEiOiJjaWtmdnA1MHAwMDN4dHdtMnBqbGR3djJxIn0.fuqVOKCu8mE-9IdxTa4R8g';
```

Comment:
This is the token that I pulled from @ayaleloehr's prototype—it might be the DSVA account